cdc/processor/pipeline/sorter.go

```diff
@@ -267,16 +280,23 @@ func (n *sorterNode) Receive(ctx pipeline.NodeContext) error {
 					model.NewResolvedPolymorphicEvent(0, n.barrierTs))
 			}
 		}
+		if n.isTableActorMode {
+			return n.sorter.TryAddEntry(ctx, msg.PolymorphicEvent)
+		}
```

Comment:
Please add some comment and a TODO the remove the flag in future.

Reply:
done